src/Tribe/Event_Tickets/Attendees_Report.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * The Events Calendar integration with Event Tickets Attendees Report class
+ *
+ * @package The Events Calendar
+ * @subpackage Event Tickets
+ * @since 4.0.1
+ */
+class Tribe__Events__Event_Tickets__Attendees_Report {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->add_hooks();
+	}
+
+	/**
+	 * Adds hooks for injecting/overriding aspects of the Attendees Report from Event Tickets
+	 *
+	 * @since 4.0.1
+	 */
+	public function add_hooks() {
+		add_action( 'tribe_tickets_attendees_event_details_list_top', array( $this, 'event_details_top' ) );
+	}
+
+	/**
+	 * Injects event meta data into the Attendees report
+	 */
+	public function event_details_top( $event_id ) {
+		if ( Tribe__Events__Main::POSTTYPE !== get_post_type( $event_id ) ) {
+			return;
+		}
+
+		$url = null;
+		if ( tribe_has_venue( $event_id ) ) {
+			$venue_id = tribe_get_venue_id( $event_id );
+
+			$url = get_post_meta( $venue_id, '_VenueURL', true );
+			if ( $url ) {
+				$url_path = parse_url( $url, PHP_URL_PATH );
```

Comment:
When using `parse_url` before PHP 5.3.3 you need `@` to avoid Warnings if it's not a valid URL.